### ISSUE TITLE
Examples on master not working with latest release

### ISSUE THREAD

**Issue by pmichel31415** [clab/dynet#1349]
Not a bug per se but as noted in https://github.com/clab/dynet/commit/f170e715e198bb166235bc3d15421f60e45921d4#commitcomment-28609161 some of the examples on master are not working with the latest release because of the recent API changes for parameters.

I think we should do a mini release including those changes so that folks who pip install the latest release don't get stuck running the examples.

This is also related to a discussion we had with @xunzhang in #1312

**Comment by zhechenyan** [comment]
@pmichel31415 Can you please tell me what exactly are the problems? I think I did not use any `dy.parameter` calls in that tutorial.

**Comment by pmichel31415** [comment]
I think the problem is that the latest release (the one you get when running `pip install dynet` does not incorporate the changes to the parameters API. There is no need to change the tutorials, just need to do a release with the `dy.parameter` removal included.

**Comment by xunzhang** [comment]
Agree, except releasing a tarball at github, we also need to update the wheel/egg files in PyPI.

**Comment by pmichel31415** [comment]
I think creating a release will automatically deploy a new egg to PyPi
